readyplayerme/rpm-unity-sdk-core
# [SDK-283]feat: centred all window content

## [SDK-283](https://ready-player-me.atlassian.net/browse/SDK-283)

## Description

-   wraps each editor window content in a "ContentWrapper" that will centre the fixed width content. Instead of it remaining left aligned when the window width expands. 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/77be31d4-bfe7-4eee-94fe-2f3a2e6e6f25)







## How to Test

-  import or checkout branch
- open all our custom editor windows (Avatar Loader, Settings, Quick start, Integration guide)
- expand the width and observe behaviour



## Checklist








[SDK-283]: https://ready-player-me.atlassian.net/browse/SDK-283?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
